### ISSUE TITLE
json-file logging

### ISSUE THREAD

**Issue by haircommander** [containers/conmon#37]
We'd like to implement a json file logger so managers have the ability to log in a json file format

**Comment by haircommander** [comment]
@phariyan PTAL

**Comment by jnovy** [comment]
@haircommander The json logger is already in the conmon for a while. Please feel free to reopen/file a new ticket if more work is needed on it.

**Comment by haircommander** [comment]
I don't believe this was done. there are currently only two recognized log types https://github.com/containers/conmon/blob/main/src/ctr_logging.c#L32-L33 that said, if there's no intention to do it, we can close this.
